frontend/src/store/slices/leafletMouse/LeafletMouseSlice.ts

```diff
@@ -1,11 +1,17 @@
 import { createAction, createSlice } from '@reduxjs/toolkit';
-import { LeafletMouseEvent } from 'leaflet';
 
-export const saveClickLatLng = createAction<LeafletMouseEvent>('saveClickLatLng');
+export const saveClickLatLng = createAction<ISerializableLeafletMouseEvent>('saveClickLatLng');
 export const clearClickLatLng = createAction('clearLatLng');
 
+export interface ISerializableLeafletMouseEvent {
+  latlng: { lat: number; lng: number };
+  originalEvent: {
+    timeStamp: number | undefined;
+  };
+}
+
 export interface ILeafletMouseSlice {
```

Comment:
Is the purpose of this slice to simply keep all the Leaflet mouse events in memory?

Reply:
the purpose of the slice is to allow other components to react to leaflet mouse events.